widget/playground/src/components/ExportConfigModal/ExportConfigModal.tsx

```diff
@@ -141,30 +141,17 @@ export function ExportConfigModal(props: ExportConfigModalProps) {
         </HelpLinksContainer>
       </Head>
       <Divider size={30} />
-
       <ButtonsContainer>
```

Comment:
The name "ButtonsContainer" is no longer suitable for this component as now it contains tabs instead of buttons.

Reply:
fixed